.github/workflows/cache_data.yaml

```diff
@@ -76,9 +76,10 @@ jobs:
 
       # Upload the downloaded files as artifacts to GitHub
       - name: Upload artifacts to GitHub
-        uses: actions/upload-artifact@v4
+        uses: actions/upload-artifact@v4.4.0
         with:
           name: gmt-cache
+          include-hidden-files: true
```

Comment:
In v4.4.0, `include-hidden-files` is `false` by default. Here we set it to `true`. 

The workflow runs as expected: https://github.com/GenericMappingTools/pygmt/actions/runs/10769413727?pr=3431